assets/dev/js/editor/editor-base.js

```diff
@@ -1359,6 +1363,10 @@ export default class EditorBase extends Marionette.Application {
 					delete controlArgs[ breakpoint + '_default' ];
 				} );
 
+				if ( deleteControlDefault ) {
```

Comment:
Do it only if there is no device default

Reply:
Done.